docs/modules/ROOT/pages/filtering.adoc

```diff
@@ -384,17 +384,12 @@ query {
 
 ==== `ID`
 
-You can only use the `_EQUAL` operator on types of `ID`.
+No Aggregation filters are available for ID.
```

Comment:
In future, I think `ID` should have the same aggregating functions as `String`. I'll jot this down!